lib/Rex/Pkg/FreeBSD.pm

```diff
@@ -69,9 +75,21 @@ sub is_installed {
   Rex::Logger::debug(
     "Checking if $pkg" . ( $version ? "-$version" : "" ) . " is installed" );
 
+  # only need -g if pkg is a glob
+  my $glob_arg = '';
+  if ( $pkg =~ /\*/ ) {
+
+    # back quote asterisks so the command can actually be used
+    $pkg =~ s/\*/\\\*/g;
+    $glob_arg = ' -g ';
+  }
+
   # pkg info -e allow get quick answer about is pkg installed or not.
   my $command =
-    $self->{commands}->{query} . " -e $pkg" . ( $version ? "-$version" : "" );
+      $self->{commands}->{query}
+    . $glob_arg
```

Comment:
For the sake of consistency, what do you think about using a similar logic for `$glob_arg` as for `$version` below? That is, only adding it to the command if needed/defined.

That might simplify the code, and we could even rename it to a more generic `$extra_args` if needed.

Reply:
Done!